### PR TITLE
Create the settings dropdown in batch edit mode

### PR DESCRIPTION
**Pull request by bp323**
Create the settings dropdown in batch edit mode. It should have the following actions:

- Edit all events
- Rename series
- Delete series

screen shot 2015-01-29 at 09 14 20

### CODE REVIEW

**Comment by bp323** [comment]
Reassigning to @Coenego for review.

File: shared/vendor/js/jquery.jeditable.js [hunk 1]
```diff
@@ -173,7 +173,7 @@
                 }
                                 
                 self.editing    = true;
-                self.revert     = $(self).html();
+                self.revert     = $(self).text();
```
Comment (bp323) [inline shared/vendor/js/jquery.jeditable.js]: This change avoids XSS issues

File: shared/vendor/js/jquery.jeditable.js [hunk 2]
```diff
@@ -312,7 +312,7 @@
                           /* check if given target is function */
                           if ($.isFunction(settings.target)) {
                               var str = settings.target.apply(self, [input.val(), settings]);
-                              $(self).html(str);
+                              $(self).text(str);
```
Comment (bp323) [inline shared/vendor/js/jquery.jeditable.js]: This change avoids XSS issues

File: shared/gh/js/controller/gh.admin-batch-edit.js [hunk 3]
```diff
@@ -98,18 +119,42 @@ define(['gh.api.series', 'gh.api.util', 'gh.api.event', 'gh.admin-constants'], f
...
             return this.revert;
         } else {
-            // If there was a change, mark the row so it's visually obvious that an edit was made to it
-            if (this.revert !== value) {
-                $('.gh-batch-edit-events-container tbody tr[data-eventid="' + $(this).closest('tr').data('eventid') + '"]').removeClass('danger active success').addClass('active');
-                // Show the save button
-                toggleSubmit();
+            // Depending on what editable field we're submitting either save it straight away or show a different save button
+            var isSeriesTitleEdit = editableField.cssclass.match('gh-jeditable-form-with-submit');
+            isSeriesTitleEdit =  isSeriesTitleEdit && isSeriesTitleEdit.length;
```
Comment (Coenego) [inline shared/gh/js/controller/gh.admin-batch-edit.js]: One space too much between `=` and `isSeriesTitleEdit `

**Comment by Coenego** [comment]
Looks good. Only left a tiny comment :) Reassigning to @bp323 for follow-up.

**Comment by bp323** [comment]
Pushed a fix, back to @Coenego for review.

**Pull request merged**